lib/flowbber/plugins/sources/lcov.py

```diff
@@ -232,21 +232,21 @@ def collect(self):
         if lcov is None:
             raise FileNotFoundError('lcov executable not found.')
 
+        # Transform from list to something like
+        #   --rc setting1=value1 --rc setting2=value2
+        rc_overrides = ''
+        if self.config.rc_overrides.value:
+            rc_overrides = '--rc {}'.format(
+                ' --rc '.join(self.config.rc_overrides.value)
```

Comment:
I think this is not ok, since rc overrides its a list. Should it contain an iterator? Maybe like this:
` rc_overrides = (
            ''.join(
                '-rc {} '.format(option)
                for option in self.config.rc_overrides.value
            )
        )`